lib/news_scraper/transformers/article.rb

```diff
@@ -20,22 +19,26 @@ def transform
         transformed_response.merge(root_domain: root_domain)
       end
 
+      def self.scrape_patterns
```

Comment:
honestly, adding a reload is an anti pattern IMHO.
This is being cached initially for article as its used in `transformed_response` in `transform` and in `scrape_details`